src/test/java/com/daml/extensions/testing/ledger/SandboxManagerIT.java

```diff
@@ -66,7 +65,9 @@ public void managerReturnsAutomaticallyAssignedLedgerId() throws Exception {
             false,
             Optional.empty(),
             Optional.empty());
-    String ledgerIdPattern = "[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}";
+
+    String ledgerIdPattern = "sandbox";
```

Comment:
Shouldn't one be able to set this when using the builder `ledgerId(...)` or something?
I mean, before, this was not a constant value.

Reply:
Yep you can do that. And there is another test dedicated to test this (SandboxIT.java).
Now ledgerId is signed inside SandboxManager.java. As I mentioned LedgerIdService set it before. Now it is deprecated so I set it to the "sandbox" rudely... I can try to assign some random character pattern to it. Or try to find Service alternative. @gaborh-da can you guide me here? what is a better solution?

Reply:
I think the documentation says that one does not need ledgerId anymore (and anywhere):
https://github.com/digital-asset/lib-daml-jvm-test/pull/75#discussion_r902461705